docs/components/bottom_app_bars.md

```diff
@@ -13,28 +13,42 @@ path: /catalog/material-bottom-app-bar/
 
 ![Example bottom app bar with inset floating action button](assets/bottom_app_bars/bottom-app-bar-hero.png)
 
+## Contents
+
+- [Bottom app bars](#bottom-app-bars)
+  - [Contents](#contents)
+  - [Using the bottom app bar](#using-the-bottom-app-bar)
+  - [Bottom app bar example](#bottom-app-bar-example)
+  - [Anatomy and key properties](#anatomy-and-key-properties)
+    - [Container attributes](#container-attributes)
+    - [FAB attributes](#fab-attributes)
+    - [Icons attributes](#icons-attributes)
+  - [Theming](#theming)
+    - [Theming example](#theming-example)
```

Comment:
We're not including this level of detail in the TOC

Reply:
Done